src/date.c

```diff
@@ -1654,7 +1613,6 @@ static void currentTimeFunc(
     sqlite3_result_text(context, zBuf, -1, SQLITE_TRANSIENT);
   }
 }
-#endif //FREEBSD_KERNEL
```

Comment:
I think this one should not be removed because it closes the `#if defined(LINUX_KERNEL_BUILD)` in line 1569

Reply:
I actually have a plain #endif there in the file now, not sure why this shows as a full removal?